setup.php

```diff
@@ -417,6 +417,27 @@ function plugin_formcreator_redirect() {
       return;
    }
 
+   if (Session::getLoginUserID() == Config::getConfigurationValue('formcreator', 'public_user_id')) {
```

Comment:
public forms now work with a user. We need to prevent him to exit from the form.